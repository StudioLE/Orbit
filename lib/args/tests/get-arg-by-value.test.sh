#!/bin/bash
set -uo pipefail

# ARGS
ACTUAL_FILE=$1

# START

OUTPUT=$(/srv/lib/args/tests/get-arg-by-value.sh \
  --string "Value of --string" \
  --int -1 \
  --flag \
  --optional value \
  -s "Value of -s" \
  -a \
  -bc \
  -- \
  /path/to/file \
  path/to/file)

echo "${OUTPUT}" > "${ACTUAL_FILE}"
