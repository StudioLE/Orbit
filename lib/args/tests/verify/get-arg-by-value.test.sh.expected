SRC ARGS COUNT: 14
REVISED ARGS COUNT: 10
--int: -1
